great_expectations/rule_based_profiler/rule/rule.py

```diff
@@ -320,3 +317,28 @@ def _get_expectation_configuration_builders_as_dict(
             expectation_configuration_builder.expectation_type: expectation_configuration_builder
             for expectation_configuration_builder in expectation_configuration_builders
         }
+
+    # noinspection PyUnusedLocal
+    @measure_execution_time(
```

Comment:
Do we want to always measure execution time, ie in a user's production environment?

Reply:
@billdirks Good point.  We can make it conditional.  I will make it a separate PR (after other changes we planned).  Thanks!